### ISSUE TITLE
videoEnabled, videoDisabled issues

### ISSUE THREAD

**Issue by hero70312** [opentok/opentok-react-native#178]
In our project, we need to build a **three people** chat room.
We want to know each stream whether it has video or not.
If user close his video, we expect to show his avatar.
So we need to know which user disable or enable his video.
 
When I user OTSubscriber and write eventHandler : videoEnabled  

```
eventHandlers={{
           videoDisabled: (event) => {
                  console.log('videoDisabled', event);
           },
          videoEnabled: (event) => {
                  console.log('videoEnabled', event);
           },
  }}
```

Event is null object, when one of user disable his video.
Can you tell me how to do get the stream's name so that I can distinguish those streams.

**Comment by msach22** [comment]
Let's keep this issue open until the fix has been implemented

**Comment by hero70312** [comment]
OK!

**Comment by msach22** [comment]
@hero70312 This should be released with `v0.9.0`

**Comment by msach22** [comment]
This has been fixed and released of `v0.9.0`
